dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ResolveDirector.java

```diff
@@ -582,9 +602,16 @@ void resolveAutoFeatures(Collection<ESAAsset> autoFeatures, ArrayList<InstallAss
             assetNamesProcessed.add(s.replaceAll("\\\\+$", ""));
         }
         Collection<ProductDefinition> productDefinitions = new HashSet<ProductDefinition>();
+        String edition = null;
         try {
             for (ProductInfo productInfo : ProductInfo.getAllProductInfo().values()) {
                 productDefinitions.add(new ProductInfoProductDefinition(productInfo));
+                edition = productInfo.getEdition();
```

Comment:
As in other comments, use productInfo.getId() to check for io.openliberty.  Don't need the "Open Liberty" or " edition." strings.